packages/repo-tools/src/commands/index.ts

```diff
@@ -58,6 +58,8 @@ function registerPackageCommand(program: Command) {
     .description(
       'Additional properties that can be passed to @openapitools/openapi-generator-cli',
     )
+    .option('--watch')
```

Comment:
I know that there is no tests for these commands, but as I haven't used them, I would really appreciate a test for the changes.

Reply:
Is there prior art for a `--watch` style test? Having a bit of trouble handling the parallel updates with jest.